docs/getting-setup.md

```diff
@@ -2,55 +2,66 @@
 
 ![][debugger-intro-gif]
 
-:construction_worker: If something goes wrong, checkout our [most common issues][common-issues] and find us in [Slack][slack]!
+### Step 1. Install Yarn
 
-### Step 1. Installing the Debugger
+```bash
+npm i -g yarn@0.16.1
```

Comment:
This seems like a very old version of Yarn to use. 1.x is stable, shouldn't we use that?